rpm_spec/core-agent.spec.in

```diff
@@ -23,6 +23,10 @@
 %define qubes_services qubes-core qubes-core-netvm qubes-core-early qubes-firewall qubes-iptables qubes-updates-proxy qubes-updates-proxy-forwarder
 %define qubes_preset_file 75-qubes-vm.preset
 
+%if 0%{?fedora} || 0%{?rhel}
+%define with_sysvinit 0
```

Comment:
The way it is written right now the build will fail if this isn't defined (non-fedora, non-rhel/centos) - because you use `%with_sysvinit` instead of `%{?with_sysvinit}`. Generally this may be a feature to force reviewing the packaging. But please leave a comment about it. Alternatively add an `%else` section defining it to 1.